sass/layouts/_about.scss

```diff
@@ -106,6 +106,18 @@
   height: 100%;
 }
 
+// Fix for firefox layer order issue
+
+.parallax-scene {
+  .layer {
+    pointer-events: none;
+  }
+  
+  #landing-register {
```

Comment:
It's impossible to select only the register button without modify html.
Unless use of :nth-last-child selector.
But the register button may not always be the last item.
Do it in that way can cause unrelated problem.